app/models/routes_into_teaching/routes_finder.rb

```diff
@@ -0,0 +1,27 @@
+module RoutesIntoTeaching
+  class RoutesFinder
+    attr_reader :routes, :answers
+
+    def initialize(answers:)
+      @routes = RoutesIntoTeaching::Route.all(route_finder: self)
```

Comment:
maybe rename `@routes` to `@all_routes` or `routes` to `recommended_routes` to avoid confusion between `@routes` and `routes` in the future?